shared/AppInsightsCore/src/JavaScriptSDK.Interfaces/ICookieMgr.ts

```diff
@@ -20,8 +20,9 @@ export interface ICookieMgr {
      * @param maxAgeSec - [optional] The maximum number of SECONDS that this cookie should survive
      * @param domain - [optional] The domain to set for the cookie
      * @param path - [optional] Path to set for the cookie, if not supplied will default to "/"
+     * @returns - True if the cookie was set otherwise false (Because cookie usage is not enabled or available)
      */
-    set(name: string, value: string, maxAgeSec?: number, domain?: string, path?: string): void;
+    set(name: string, value: string, maxAgeSec?: number, domain?: string, path?: string): boolean;
```

Comment:
While this is a change to the signature of the function, it should be backward compatible as previously it was void.

Reply:
This is also used by internally as well, so this should be compatible with that as well.